src/simplelib.c

```diff
@@ -0,0 +1,90 @@
+#include <string.h>
+
+void *memcpy(void *destination, const void *source, size_t num) {
```

Comment:
Wasm provides `memcpy` and `memfill` bulk-memory opcodes we could instead codegen which should be faster & cheaper